src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -62,9 +62,17 @@ public function getEnvPlaceholders()
 
     /**
      * Merges the env placeholders of another EnvPlaceholderParameterBag.
+     *
+     * @param EnvPlaceholderParameterBag $bag
```

Comment:
We generally prefer removing those, eventhough the code base is not always consistent on this aspect.